0_locs_poly_setup/src/points_to_csv.R

```diff
@@ -0,0 +1,22 @@
+#' Function to convert a {sf} of points to a tibble, retaining the Latitude and 
```

Comment:
question: Should this say "points to a csv", not tibble?

Reply:
Absolutely should! Thank you